portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,24 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.List;
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some data. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
-  }
+	@Override
+	public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+		List<String> msgs = new ArrayList<>();
+		msgs.add("hello");
+		msgs.add("goodbye");
+		msgs.add("see ya");
+		Gson gson = new Gson();
+		String json = gson.toJson(msgs);
+    // Send the JSON as the response
+		response.setContentType("application/json;");
+		response.getWriter().println(json);
+	}
```

Comment:
spacing

Reply:
oh gosh... it looks okay on my end. I specifically made a commit to change that but it still seems wrong